docs/about/model.md

```diff
@@ -35,7 +35,7 @@ All bids will be processed by the auction smart contract , and all of the HEZ to
 - 30% will be allocated as Hermez Network usage incentives, compensating active engagement and network adoption, e.g. rewarding transaction and rewarding the holding of specific tokens in Hermez L2 addresses, instead of on L1 Ethereum addresses.
 
 
-The auction process incentivises efficiency, as coordinators need to include as many transactions as possible in each time slot to compensate for their bidding costs and operative expenses.
+The auction process incentivizes efficiency, as coordinators need to include as many transactions as possible in each time slot to compensate for their bidding costs and operative expenses.
```

Comment:
DS mentioned he wanted to use British spelling, so we could keep it "incentivises"